tasks.mak

```diff
@@ -109,15 +107,17 @@ endif
 	mkdir -p ./confgenerator/testdata/goldens/$(TEST_NAME)/golden
 	touch ./confgenerator/testdata/goldens/$(TEST_NAME)/input.yaml
 
-dist/fluent-bit:
+dist/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit:
 	$(MAKE) fluent_bit_local
 
-dist/otelopscol:
+dist/opt/google-cloud-ops-agent/subagents/opentelemetry-collector/otelopscol:
 	$(MAKE) otelopscol_local
 
 .PHONY: transformation_test
-transformation_test: dist/fluent-bit dist/otelopscol
-	FLB=$(PWD)/dist/fluent-bit OTELOPSCOL=$(PWD)/dist/otelopscol go test ./transformation_test $(if $(UPDATE),-update,)
+transformation_test: dist/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit dist/opt/google-cloud-ops-agent/subagents/opentelemetry-collector/otelopscol
+	FLB=$(PWD)/dist/opt/google-cloud-ops-agent/subagents/fluent-bit/bin/fluent-bit \
+	OTELOPSCOL=$(PWD)/dist/opt/google-cloud-ops-agent/subagents/opentelemetry-collector/otelopscol \
```

Comment:
[Optional] Let's use `$${PWD}` (bash variable quoting) instead of `$(PWD)` (makefile quoting)?

Reply:
Why wouldn't we want to use makefile quoting in the makefile?

Reply:
In this case, I do want the working directory of make and not the working directory of the shell, although I do not change the working directory of the shell during this target.